### DISCUSSION TITLE
I've got problems with the Create a release based workflow

### DISCUSSION THREAD

**Discussion by rfalanga** [skills/.github#67] [Q&A]
I've got some real problems with the Skills _Create a release-based workflow_ module.

# In Step 1
Step 1, number 2 says, "... click *0 releases*".  There is no "0 releases" link.

# Step 2 has problems
In Step 2 I'm to change the background color to black in the base.css file. But it also says that the text color is to be changed to green and, "I'll help you change the text color to green". Huh? When? Where? The instructions don't show me that the text color is going to be green. Does that happen later? The wording suggests that it might be later, but I'm not sure. Or has that already been changed? If it has already been changed, then why doesn't it say, "I have changed the text color to green". 

I'm finding the instructions in this module to be too ambivalent.

# Step 3
At the end of step 3 I wasn't sure if I was supposed to do. Was I supposed to complete the PR or merge? It says wait 20 seconds then refresh the page. I waited, then refreshed the page. No change. Nothing is different. I don't get it.

# Onto Step 4 problems 
By now things have gotten pretty far off.  I've got the _Release v1.0_ PR open, in the "Activity Merge into main" section. Its instructions me to click on the "Merge pull request" button. There is no "Merge pull request" button. There's a delete draft button. There's a Publish button, but **there is no Merge pull request button**!!!

I don't know what to do now. I'm giving up. This module has WAY too many assumptions that the user will know what the author of Create a release based workflow is asking the user to do. I'm sorry, but I cannot read minds. I need explicit instructions, because if there is any equivocal language and I'm left to my own to guess what I'm supposed to do, I will generate lots of possible paths to follow, none of which I'm sure will work.

# What I will do now
I'm going to wait for your reply. I will blow away the repo I've created on this because I've got it into such a miserable state that I consider it useless, at least to help me understand the stated topic of creating a release based workflow. I look forward to hearing back from you and correcting the missteps I've taken.

**Comment by sinsukehlab** [comment]
Here is my answer.
I hope it will help you to try again.

# Step 1
The instruction should be the same as Step 4 number 1.
# Step 2
@github-actions bot will do the rest when you finish the Activity.
# Step 3
You should open a pull request comparing the `release-v1.0` branch to the `main` branch.
# Step 4
You should finish Step 3. You haven't opened the pull request.

**Reply by sinsukehlab** [reply]
I have added a commit to my PR to resolve the issue on Step 1.
Cf. skills/release-based-workflow#9

**Reply by rfalanga** [reply]
Thank you very much for your explanation, @sinsukehlab. I'll get rid of the repo I created when starting this training module, then follow both the training module and your instructions

**Reply by sinsukehlab** [reply]
I'm glad to help you.
I would appreciate it if you could mark this as an answer.

**Comment by The11sun** [comment]
I don't know anything..if I miss the steps. or by any means The truth is, I
don't know.
sorry in advance
   I'm still dizzy right now, what stage am I in? I'm still wondering

ในวันที่ พฤ. 6 ต.ค. 2022 08:16 น. Rod Falanga ***@***.***>
เขียนว่า:
